### ISSUE TITLE
Update deps

### ISSUE THREAD

**Issue by fundon** [balderdashy/sails#2966]
```bash
$ npm outdated --depth=0 --long                     [master±‣ 1]
Package               Current  Wanted  Latest  Location              Package Type
grunt-contrib-uglify    0.4.1   0.4.1   0.9.1  grunt-contrib-uglify  dependencies
grunt-sync              0.0.8   0.0.8   0.2.3  grunt-sync            dependencies
ejs                     0.8.8   0.8.8   2.3.1  ejs                   dependencies
connect-redis           1.4.5   1.4.5   2.3.0  connect-redis         dependencies
grunt-contrib-cssmin    0.9.0   0.9.0  0.12.3  grunt-contrib-cssmin  dependencies
grunt-sails-linker      0.9.6   0.9.6  0.10.1  grunt-sails-linker    dependencies
grunt-contrib-watch     0.5.3   0.5.3   0.6.1  grunt-contrib-watch   dependencies
grunt-contrib-concat    0.3.0   0.3.0   0.5.1  grunt-contrib-concat  dependencies
grunt                   0.4.2   0.4.2   0.4.5  grunt                 dependencies
grunt-contrib-less     0.11.1  0.11.1   1.0.1  grunt-contrib-less    dependencies
rc                      0.5.5   0.5.5   1.0.3  rc                    dependencies
grunt-contrib-copy      0.5.0   0.5.0   0.8.0  grunt-contrib-copy    dependencies
sails                  0.11.0  0.11.1  0.11.0  sails                 dependencies
grunt-contrib-clean     0.5.0   0.5.0   0.6.0  grunt-contrib-clean   dependencies
grunt-contrib-coffee   0.10.1  0.10.1  0.13.0  grunt-contrib-coffee  dependencies

```

**Comment by sailsbot** [comment]
Thanks for posting, @fundon.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
